detectable/src/main/java/com/blackduck/integration/detectable/detectables/gradle/inspection/GradleInspectorExtractor.java

```diff
@@ -66,12 +67,21 @@ public Extraction extract(File directory, ExecutableTarget gradleExe, @Nullable
             reportFiles.toArray(files);
             List<File> reportFilesSorted = Arrays.asList(sortFilesByDepth(files));
 
-            reportFilesSorted.stream()
-                .map(gradleReportParser::parseReport)
-                .filter(Optional::isPresent)
-                .map(Optional::get)
-                .map(gradleReportTransformer::transform)
-                .forEach(codeLocations::add);
+            if (rootOnly && reportFilesSorted.size()>0) {
+                logger.debug("Gradle Inspector root-only option selected. Only processing root project dependencies.");
```

Comment:
Only root project dependencies will be processed by the Gradle Inspector as per user selection.

Reply:
Will keep current wording as approved by Chris but thank you for the suggestion